experiment/get_job_pods.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script prints out lines with: "job-name: ['pod-id', 'pod-id-2']""
+#
+# USAGE: have KUBECONFIG pointed at your prow builds cluster then:
+#
+# get_job_pods.py [--show-all] 
+#
+# EG:
+# `experiment/get_job_pods.py --show-all | grep pull-kubernetes-bazel-test`
```

Comment:
maybe make job as an arg, `experiment/get_job_pods.py --show-all --job=pull-kubernetes-bazel-test` and output `['c9e634a5-cbe6-11e7-9149-0a580a6c0216']`, which will make it easier to use for other scripts want to reference this.

Reply:
`deck` is pretty good for grabbing pods by a particular job, but not if we hide it 😬

Reply:
or maybe optional `--filter=whatever-regex`? grep is not part of python though

Reply:
Hmm, I think other scripts should just import or copy the python anyhow actually or even better: write something less hacky.
I think we should keep this script to the original use case, which is quickly getting jobs -> pod names for debugging prow jobs.